test/goss/goss.yaml

```diff
@@ -0,0 +1,39 @@
+# GOSS Manual: https://github.com/aelsabbahy/goss/blob/master/docs/manual.md
+# CMD: GOSS_FILES_PATH=./test/goss dgoss run -i -v $PWD:/project ppiper/mta-archive-builder
+package:
+  git:
+    installed: true
+    versions:
+    - 1:2.11.0-3+deb9u4
+  python-minimal:
```

Comment:
because https://github.com/SAP/devops-docker-images/issues/116 and https://github.com/SAP/devops-docker-images/issues/107 I would prefer to test for the ‘full’ python here

Reply:
fixed